main.go

```diff
@@ -262,9 +262,23 @@ func createStartBatchRunParams(cfg Config, appFileNumber int) map[string]interfa
 	switch cfg.AppType {
 	case "app_file":
 		params["app_file_number"] = appFileNumber
+		if cfg.OsName == "ios" {
+    	    if cfg.Environment == "remote_testkit" {
+    			params["bundle_id"] = cfg.BundleID
+    	    } else if cfg.Environment == "remote_testkit_onpremise" {
+    			params["bundle_id"] = cfg.BundleID
+    	    }
+		}
```

Comment:
Code indent is strange in some parts. I'd appreciate if you follow existing codes.

Reply:
Sorry for pointing out such a fundamental mistake. It looks like a problem caused by a configuration of IDE or editors, etc. What kind of IDE or editors do you use?

Reply:
@nkns165

Reply:
I'm using Visual Studio Code and Go plugin (https://github.com/microsoft/vscode-go).
Maybe it's also due to tab characters (sorry I've not configured much).

I will fix it on my side this time.

Reply:
Actually I also use that set. But I also have vim plugin so maybe because of the vim plugin. Let me check as well

Reply:
After disabling vim plugin, it looks fine. It looks like no problem. Shall I fix it?

Reply:
Yes, please go ahead